docs/explanations/audit-scanner.md

```diff
@@ -0,0 +1,273 @@
+section provides a summary of the audit results, showing that there were no
+errors, failures, or warnings.
+
+```yaml
+apiVersion: wgpolicyk8s.io/v1beta1
+kind: ClusterPolicyReport
+metadata:
+  creationTimestamp: "2023-07-10T19:25:40Z"
+  generation: 1
+  labels:
+    app.kubernetes.io/managed-by: kubewarden
+  ...
+results:
+- policy: cap-testing-cap-policy
+  ...
+  resourceSelector: {}
+  resources:
+  - apiVersion: v1
+    kind: Namespace
+    name: kube-system
+    ...
+  result: pass
+  rule: testing-cap-policy
+  source: kubewarden
+  timestamp:
+    nanos: 0
+    seconds: 1689017140
+- policy: cap-testing-cap-policy
+  ...
+  resourceSelector: {}
+  resources:
+  - apiVersion: v1
+    kind: Namespace
+    name: default
+    ...
+  result: pass
+  rule: testing-cap-policy
+  source: kubewarden
+  timestamp:
+    nanos: 0
+    seconds: 1689017140
+...
+summary:
+  error: 0
+  fail: 0
+  pass: 6
+  skip: 0
+  warn: 0
+```
+
+
+### Namespace-Specific Audit Results example
+
+In this example, the audit scanner has evaluated multiple policies on resources
+within the `default` namespace. The results indicate that some of the resources
+failed the validation for the `cap-no-privilege-escalation` policy, while
+others passed the validation for the `cap-do-not-run-as-root` policy. The
+`summary` section shows a summary of the audit results, indicating the number
+of failures and passes.
+
+```yaml
+apiVersion: wgpolicyk8s.io/v1beta1
+kind: PolicyReport
+metadata:
+  creationTimestamp: "2023-07-10T19:28:05Z"
+  generation: 4
+  labels:
+    app.kubernetes.io/managed-by: kubewarden
+  ...
+results:
+- message: one of the containers has privilege escalation enabled
+  policy: cap-no-privilege-escalation
+  ...
+  resourceSelector: {}
+  resources:
+  - apiVersion: apps/v1
+    kind: Deployment
+    name: nginx-deployment
+    namespace: default
+    ...
+  result: fail
+  rule: no-privilege-escalation
+  source: kubewarden
+  timestamp:
+    nanos: 0
+    seconds: 1689017383
+- policy: cap-do-not-run-as-root
+  ...
+  resourceSelector: {}
+  resources:
+  - apiVersion: apps/v1
+    kind: Deployment
+    name: nginx-deployment
+    namespace: default
+    ...
+  result: pass
+  rule: do-not-run-as-root
+  source: kubewarden
+  timestamp:
+    nanos: 0
+    seconds: 1689017383
+...
+summary:
+  error: 0
+  fail: 8
+  pass: 10
+  skip: 0
+  warn: 0
+```
+
+
+By using the audit scanner and analyzing the generated audit reports, you can
+gain insights into the compliance of your resources with the deployed policies,
+helping you ensure that your cluster remains secure and adheres to the defined
+rules and validations.
+
+---
+
+## Policies
+
+Policies will be evaluated by the audit scanner by default. Operators that want to
+skip a policy evaluation in the Audit scanner should set `false` to the
+`backgroundAudit` field in the policy spec. Furthermore,  policies in
+Kubewarden now support two optional annotations:
```

Comment:
I think this should be refactored to not sound like release notes but evergreen info.
Something like:

> The Audit Scanner also uses the following policy `metadata.annotations` in the created PolicyReport results:
> - `io.kubewarden.policy.severity`: One of (...). The Audit Scanner will always record the result as `info` severity if the policy was deployed in monitor mode.
> - `io.kubewarden.policy.category:` (...)

Now that I think about it, we shouldn't mention `metadata.yaml` here, that's only for policy writers. Policy users just consume the annotations from `kwctl scaffold` or change them.